### DISCUSSION TITLE
Grafana alert cannot get the log content related to the alert keyword

### DISCUSSION THREAD

**Discussion by weiwuduzui** [grafana/grafana#68958] [Alerting]
Our log alarm rules use a unified method, such as: warning, error, etc. If you cannot see the specific log content in the alarm, you will not be able to obtain the specific warning or error at that time

example:
hello warning
hi warning

If only the keyword warning is retrieved, it is impossible to distinguish which alarm is

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82703. Feel free to continue the discussion around this there. Thank you!
